src/drive/mobile/lib/cozy-helper.js

```diff
@@ -19,7 +19,7 @@ export const initClient = url => {
       'io.cozy.jobs:POST:sendmail:worker'
     ],
     oauth: {
-      redirectURI: 'http://localhost',
+      redirectURI: 'cozydrive://auth',
```

Comment:
Don't we need some new code to handle this deeplink?

Reply:
No need :
- to handle deeplink. It’s already done somewhere ;) 
- migrating to cozy-auth will come soon. But as it needs more work and as is only a technical task I prefer to do it in a new PR.